react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Stopwatch extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      isRunning: false,
+      time: 0
+    };
+    this.handleClick = this.handleClick.bind(this);
+    this.pressPlayOrPause = this.pressPlayOrPause.bind(this);
```

Comment:
I could be mistaken, but looking at your code, I don't think there's a reason to bind `this.pressPlayOrPause`.